app/src/pages/inside/logsPage/logsGrid/attachmentBlock/attachmentBlock.jsx

```diff
@@ -39,14 +47,19 @@ export class AttachmentBlock extends Component {
     const {
       value,
       customProps: { consoleView },
+      tokenValue,
     } = this.props;
 
     return (
       <div className={cx('attachment-block')} onClick={this.onClickAttachment}>
         {consoleView ? (
           <div className={cx('image', 'console-view')}>{Parser(AttachIcon)}</div>
         ) : (
-          <img className={cx('image')} src={getFileIconSource(value)} alt={value.contentType} />
+          <img
```

Comment:
Please use `Image` component and remove token from url - it will be sent in header

Reply:
Fixed